packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarVirtual.tsx

```diff
@@ -36,7 +36,7 @@ export default function ServerSidebarVirtual({
       } else {
         item.isOpen = !item.isOpen;
       }
-      if (item.href) {
+      if (item.href && !item.isExternal) {
```

Comment:
what about all the items that have an href and are not flagged external ? won't this the navigation for a lot of items in the sidebar ?

Reply:
For non external links, the behavior is equal to currently, nothing changed. There is a bug for external links, even tough a new tab is opened, the link is also loaded inside the manager app. 
see `window.location.href = item.href`